bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/ServiceBean.java

```diff
@@ -17,5 +17,13 @@
     ServiceBean() {
     }
 
+    /**
+     * Service Name
+     */
     public String name;
+
+    /**
+     * Persistence service class
+     */
+    public String classname;
```

Comment:
Exposing a service name (ie label) is also something I was thinking about - if you're ok, I'll add a 'getLabel()' method to the PersistenceService class?